### PR TITLE
model-conversion : make causal-verify-logits fails with model names containing "."

### PR DESCRIPTION
**Pull request by DamonFool**
`make causal-verify-logits` fails if model names containing "." like `Qwen2.5-0.5B-Instruct`.
```bash
export MODEL_PATH=~/share/Qwen2.5-0.5B-Instruct
make causal-verify-logits
```

The following is the error msg.
```
Logits saved to data/llamacpp-Qwen2.5-0.5B-Instruct.bin
Logits saved to data/llamacpp-Qwen2.5-0.5B-Instruct.txt
ggml_metal_free: deallocating
Error: PyTorch logits file not found: data/pytorch-Qwen2.5-0.bin
Please run scripts/run-org-model.sh first to generate this file.
make: *** [causal-verify-logits] Error 1
```

It should be `data/pytorch-Qwen2.5-0.5B-Instruct.bin` instead of `data/pytorch-Qwen2.5-0.bin`.

I suggest just using `os.path.basename(model_path)` to fix it assuming that the `model_path` should be a dir path.

### CODE REVIEW

**Review by danbev — APPROVED** [review]

**Comment by danbev** [comment]
@DamonFool Thanks again!

**Comment by DamonFool** [comment]
@danbev Thanks again too.
